profiles/base-empty.yaml

```diff
@@ -96,4 +96,5 @@ empty_values:
   hollow_eve_loot_container: []
   wands: []
   wand_watcher_no_use_scripts: []
-  wand_watcher_no_use_rooms: []
\ No newline at end of file
+  wand_watcher_no_use_rooms: []
+  noheal_empathylink: []
```

Comment:
This should _also_ be in `base.yaml`

Reply:
I think I did put in base but forgot to add to her. Let me check. :smile:

Reply:
Okay, added and committed to the files.